installing/installing-preparing.adoc

```diff
@@ -39,7 +39,7 @@ endif::openshift-origin[]
 
 You can deploy an {product-title} 4 cluster to both on-premise hardware and to cloud hosting services, but all of the machines in a cluster must be in the same data center or cloud hosting service.
```

Comment:
I think this is a better place for the new adding. 
{product-title} does not support cloud-based storage for bare metal installation.

@jsafrane Do you think this sentence make sense? Or any suggestion here?

Reply:
Yes, this place sounds a bit better to me, but it's not much of a difference.